mapper/helper.go

```diff
@@ -11,3 +20,30 @@ func EqualFoldContains(arr []string, str string) bool {
 	}
 	return false
 }
+
+// IsPChain checks network identifier to make sure sub-network identifier set to "P"
+func IsPChain(networkIdentifier *types.NetworkIdentifier) bool {
+	return networkIdentifier != nil &&
+		networkIdentifier.SubNetworkIdentifier != nil &&
+		networkIdentifier.SubNetworkIdentifier.Network == PChainNetworkIdentifier
+}
+
+// GetAliasAndHRP fetches chain id alias and hrp for address formatting.
+// Right now only P chain id alias is supported
+func GetAliasAndHRP(networkIdentifier *types.NetworkIdentifier) (string, string, error) {
+	if !IsPChain(networkIdentifier) {
+		return "", "", errUnsupportedChain
+	}
+
+	var hrp string
+	switch networkIdentifier.Network {
+	case FujiNetwork:
+		hrp = constants.GetHRP(constants.FujiID)
+	case MainnetNetwork:
+		hrp = constants.GetHRP(constants.MainnetID)
+	default:
+		return "", "", errUnsupportedNetwork
+	}
+
+	return PChainIDAlias, hrp, nil
```

Comment:
I think we can remove `PChainIDAlias` and use `PChainNetworkIdentifier` since we're using the shorthand `P`, `X`, and `C` for the identifiers.